src/Microsoft.Azure.SignalR/ServiceOptionsSetup.cs

```diff
@@ -70,14 +76,14 @@ private TimeSpan GetConfiguredTimeSpanFromSeconds(int? seconds, TimeSpan default
             return defaultValue;
         }
 
-        private (string AppName, string ConnectionString, ServiceEndpoint[] Endpoints, ConfigurableServiceOptions configurableOptions) ParseConfiguration()
+        private (string AppName, string ConnectionString, ServiceEndpoint[] Endpoints, ConfigurableServiceOptions configurableOptions) ParseConfiguration(string connectionName)
         {
-            var sectionKey = Constants.Keys.AzureSignalRSectionKey;
+            var sectionKey = string.IsNullOrEmpty(connectionName) ? Constants.Keys.AzureSignalRSectionKey : $"{Constants.Keys.AzureSignalRSectionKey}:{connectionName}";
```

Comment:
An empty `connectionName` should be treated as `Azure:SignalR:`? `Azure:SignalR:` is also valid section name.

Reply:
empty connectionName is considered as the default case, added a null check for AddAzureSignalRWithConnectionName